### PR TITLE
feat(client-go): add builds endpoint

### PR DESCRIPTION
**Pull request by Joshua-Anderson**

### CODE REVIEW

File: client-go/cmd/builds.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
+package cmd
+
+import (
+	"fmt"
+
+	"gopkg.in/yaml.v2"
```
Comment (mboersma) [inline client-go/cmd/builds.go]: I was going to squawk about multiple vendoring techniques here, but I see we [already import this way](https://github.com/deis/deis/blob/master/builder/utils.go#L7) in the builder. We could use `godep` to vendor from `gopkg.in` for consistency, but I'm curious if other maintainers see this as an issue.

Comment (Joshua-Anderson) [inline client-go/cmd/builds.go]: It is already in godeps, that's why I'm using it :smiley:.

Comment (mboersma) [inline client-go/cmd/builds.go]: I should have looked! Nevermind, problem solved.

**Comment by mboersma** [comment]
Code LGTM.

**Comment by technosophos** [comment]
Code LGTM

**Comment by Joshua-Anderson** [comment]
Rebased!

**Pull request merged**